README.md

```diff
@@ -36,6 +36,15 @@ The following repositories describe the underlying API and controllers for the a
 
 There are demonstrations available at https://github.com/open-cluster-management/demo-subscription-gitops .
 
+### Governance, Risk and Compliance (GRC) across Kubernetes clusters
+
+* Use prebuilt security and configuration controllers to enforce policy on Kubernetes configuration, identity and access management (IAM), Center for Internet Security (CIS), and certificate management across your clusters.
+* Use the governance and risk dashboard to view and manage the number of security risks and policy violations in all of your clusters and applications.
```

Comment:
should we remove the dashboard? it is open sourced but still not quite ready to use for ocm project.

Reply:
Ok. Let me know if you have some other updates as well. I just took the old description and modified a little bit because it was out of date. Maybe can we replace the console bullet with a sentence about the policy framework which didn't exist back then. Thanks.